src/shared/copy.c

```diff
@@ -850,7 +847,7 @@ static int fd_copy_fifo(
         if (copy_flags & COPY_MAC_CREATE)
                 mac_selinux_create_file_clear();
         if (FLAGS_SET(copy_flags, COPY_GRACEFUL_WARN) && (ERRNO_IS_NEG_PRIVILEGE(r) || ERRNO_IS_NEG_NOT_SUPPORTED(r))) {
-                log_notice_errno(r, "Failed to copy fifo '%s', ignoring: %m", from);
+                log_notice_errno(r, "Failed to copy fifo '%s', ignoring: %m", strna(from));
```

Comment:
I'd prefer that we skip the filename completely if we don't have it instead of `n/a`, but doesn't matter too much